compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/DesktopPopup.desktop.kt

```diff
@@ -44,15 +42,8 @@ fun rememberCursorPositionProvider(
     alignment: Alignment = Alignment.BottomEnd,
     windowMargin: Dp = 4.dp
 ): PopupPositionProvider = with(LocalDensity.current) {
-    val component = LocalLayerContainer.current
-    val cursorPoint = remember {
-        val awtMousePosition = MouseInfo.getPointerInfo().location
-        convertPointFromScreen(awtMousePosition, component)
-        IntOffset(
-            (awtMousePosition.x * component.density.density).toInt(),
-            (awtMousePosition.y * component.density.density).toInt()
-        )
-    }
+    val pointerPositions = LocalPointerPositions.current
+    val cursorPoint = remember { pointerPositions.entries.first().value.round() }
```

Comment:
In the future when @igordmn will merge his changes about multi-touch support, we can filter by mouse here.